lib/enterprise-event-stream.js

```diff
@@ -0,0 +1,204 @@
+/**
+ * @fileoverview Enterprise event stream backed by the enterprise events API
+ */
+
+'use strict';
+
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+
+const Readable = require('stream').Readable,
+	util = require('util');
+
+// ------------------------------------------------------------------------------
+// Private
+// ------------------------------------------------------------------------------
+
+/** @const {int} The default polling interval (in seconds) */
+const DEFAULT_POLLING_INTERVAL = 60;
+
+/** @const {int} The default number of events to fetch in each poll */
+const DEFAULT_CHUNK_SIZE = 500;
+
+// ------------------------------------------------------------------------------
+// Public
+// ------------------------------------------------------------------------------
+
+/**
+ * Stream of Box enterprise events.
+ *
+ * By default, the stream starts from the current time.
+ * Pass 'startDate' to start from a specific time.
+ * Pass 'streamPosition' to start from a previous stream position, or '0' for all available past events (~1 year).
+ * If pollingInterval = 0, then the stream will end when all available events have been delivered (no polling).
```

Comment:
This seems like a weird option, since there are already stream flow control options; why would a user want a stream that only makes one call?